src/backend/src/controllers/work-packages.controllers.ts

```diff
@@ -101,10 +101,16 @@ export const createWorkPackage = async (req: Request, res: Response) => {
   // and what number work package this should be
   const { carNumber, projectNumber, workPackageNumber } = projectWbsNum;
 
-  if (workPackageNumber !== 0) throw new TypeError('Given WBS Number is not for a project.');
+  if (workPackageNumber !== 0) {
+    return res
+      .status(400)
+      .json({ message: `Given WBS Number ${projectWbsNum.toString()} is not for a project.` });
```

Comment:
this will just print `Given WbS Number [object Object] is not for a project.` -- instead do something like `Given WBS Number ${carNumber}.${projectNumber} ...` blah blah blah etc yk